test/persistence/persistence.jl

```diff
@@ -1,203 +1,30 @@
 @testset "Persistence" begin
-    pdict = load(joinpath(testdir,"testdata","tutte-pathdigraph.jgz"))
+    @test_throws ErrorException LightGraphs._NI("Not implemented")
+    pdict = loadgraphs(joinpath(testdir,"testdata","tutte-pathdigraph.jgz"))
     p1 = pdict["Tutte"]
     p2 = pdict["pathdigraph"]
     g3 = PathGraph(5)
 
-    function readback_test(format::Symbol, g::Graph, gname="g",
```

Comment:
We weren't using it before for LGFormat; I didn't change any of the tests except to ensure 100% coverage.